addon/components/sl-date-time.js

```diff
@@ -84,7 +84,7 @@ export default Ember.Component.extend( TooltipEnabled, {
     /**
      * The bound value of the component's date value
      *
-     * @type {Date}
+     * @type {*}
```

Comment:
Change to `@type {Array|Date|moment|Number|Object|String|undefined}` per the types supported at http://momentjs.com/docs/#/parsing/